jwst/straylight/straylight_step.py

```diff
@@ -14,91 +14,36 @@ class StraylightStep (Step):
 
     class_alias = "straylight"
 
-    spec = """
-         method = option('Nearest','ModShepard',default='ModShepard') #Algorithm method
-         roi = integer(2, 1024, default = 50) # Region of interest given as even integer
-         power = float(0.1, 5, default = 1.0) # Power of weighting function
-
-    """
-
-    reference_file_types = ['regions']
+    reference_file_types = ['mrsxartcorr']
```

Comment:
Should update the doc page we have somewhere in jwst/docs/jwst that lists the ref file types used by each step

Reply:
Found some pages at docs/jwst/references_general/ that looked like it could use some doc pages by analogy to other steps, so created them.  Let me know if anywhere else specific is missing.